util-images/network/netperfbenchmark/Makefile

```diff
@@ -0,0 +1,17 @@
+PROJECT = k8s-testimages
+IMG = gcr.io/$(PROJECT)/netperfbenchmark
+TAG = 0.01
+
+all: push
+
+.PHONY: build
+build: cmd
+	docker build --pull -t $(IMG):$(TAG) .
+	docker tag $(IMG):$(TAG) $(IMG):latest
+	@echo Built $(IMG):$(TAG) and tagged with latest
+
+.PHONY: push
+push: build
+	docker push $(IMG):$(TAG)
+	docker push $(IMG):latest
```

Comment:
Do we really need latest? Let's not push latest for now (same for build).

Reply:
Modified